pkg/drivers/kic/kic.go

```diff
@@ -220,8 +220,7 @@ func (d *Driver) GetURL() (string, error) {
 
 // GetState returns the state that the host is in (running, stopped, etc)
 func (d *Driver) GetState() (state.State, error) {
-	// allow no more than 2 seconds for this. when this takes long this means deadline passed
-	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), 8*time.Second)
```

Comment:
any chance we could still warn the user to restart their docker, when it takes that long ?
in 90% of the times if inspect takes too much time, restarting docker makes it super fast.

Reply:
Will add a warning.